src/app/components/MediaLoader/utils/getProducerFromServiceName.ts

```diff
@@ -0,0 +1,21 @@
+// The "producer" we report is sometimes different to the service name
+// http://bbc.github.io/echo-docs/pages/reporting-piano.html#valid-producer-values
+const getProducerFromServiceName = (serviceName: string | null) => {
+  if (typeof serviceName !== 'string') return '';
+
+  const replacementProducerName = {
```

Comment:
Ello, was wondering if the other services will be added later on too?

Reply:
Ooh what do you mean? I think all the services [in this list](http://bbc.github.io/echo-docs/pages/reporting-piano.html#valid-producer-values) should be accounted for.

E.g. if the service name matches the producer name then it will be made uppercase e.g. PIDGIN

However if the service name does not match the producer name then it will be taken from this list e.g. turkce -> TURKISH

Or have I missed something?

Reply:
Oh wicked cool.

Reply:
I didn't know the other services hadn't been accounted for, soz 
😅